modules/knative-serving-CR-system-deployments.adoc

```diff
@@ -0,0 +1,40 @@
+[id="knative-serving-CR-system-deployments_{context}"]
+= Overriding system deployment configurations
+
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR).
+
+Currently, overriding default configuration settings for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields are supported.
```

Comment:
Could say: "You can overide default configuration settings for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields."

Reply:
I think since the previous line starts with "You can override..." it reads oddly to have this twice, so I'll leave it as is.